### DISCUSSION TITLE
pre-rfc: Add log channels

### DISCUSSION THREAD

**Discussion by Xuanwo** [databendlabs/databend#7010] [Ideas]
We used `query_enabled` to control whether we print query-related logs. But in RFC https://databend.rs/doc/contributing/rfcs/new-logging, we removed it. Instead, we unified all our logs to the same logger.

However, based on discussion with @everpcpc, their does have the need for different log channels:

- system log: how query/metasrv works internally
- user log: user's request/response, query content, etc for audit

Maybe we can introduce the concept of channels for users to config like crdb dos: https://www.cockroachlabs.com/docs/stable/logging-overview.html

## Information I needed

- Channels that are needed inside the databend-query.
- Should the channel be non-overlapping? Do we need to have a channel that contains all logs?

**Comment by Xuanwo** [comment]
Cc @flaneur2020 @ZhiHanZ @everpcpc for more input.

**Comment by everpcpc** [comment]
I prefer the non-overlapping way.

**Comment by flaneur2020** [comment]
a non-overlapping way +1
